package.json

```diff
@@ -52,6 +58,9 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
+  "dependencies": {
+    "@expo/config-plugins": "^4.0.4"
```

Comment:
probably should be an optional peer dependency?

see eg https://github.com/rnmods/react-native-document-picker/blob/d2647fec02f926a8482a12667e2ff63d7852f2d4/package.json#L83

Reply:
Thank your for your review. Let's me update it.

Reply:
or maybe just dev dependency? not sure here